DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs

```diff
@@ -991,15 +598,15 @@ async Task UpdateCommandPermissionGroup(GroupCommand groupCommand)
                                 com.CommandId = command.Id;
 
                                 var source = commandTypeSources.FirstOrDefault(f => f.Key == com.Method.DeclaringType);
-                                await UpdateCommandPermission(command.Id, com.Name, source.Value, source.Key);
+                                await PermissionWorker.UpdateCommandPermissionAsync(types, guildid, command.Id, com.Name, source.Value, source.Key);
                             }
 
                             else if (groupCommands.Any(x => x.Name == command.Name))
                             {
                                 var com = groupCommands.First(x => x.Name == command.Name);
```

Comment:
similar to earlier comments -- we're looking for the same thing twice here. Should only need to do this once

Reply:
eh?